From: Adrian Knoth <adi@drcomp.erfurt.thur.de>
Forwarded: not-needed
Origin: https://github.com/jackaudio/jack2/commit/871bd0851d7a300c5c159b3ad42bad95ab6d97aa
Description: Fix naming of inprocess.so
 This fix has been incorporated upstream, so drop it on next import.
--- a/example-clients/wscript
+++ b/example-clients/wscript
@@ -149,7 +149,7 @@ def build(bld):
 
     for example_lib, example_lib_source in list(example_libs.items()):
         lib = bld(features = 'c cshlib')
-        lib.env['shlib_PATTERN'] = '%s.so'
+        lib.env['cshlib_PATTERN'] = '%s.so'
         lib.includes = os_incdir + ['../common/jack', '../common']
         lib.target = example_lib
         lib.source = example_lib_source
